src/components/props/getExpressions.js

```diff
@@ -10,8 +10,13 @@ import extractExpressionName from './extractPropName'
  * @returns {Object|string|null}
  */
 export function extractExpressions (exprType, attributes) {
-  const objectExprKey = exprType === 'props' ? 'vprops' : 'vdata'
-  const objectPropExprRegExp = exprType === 'props' ? /vprops/i : /vdata/i
+  const exprKeys = {
+    props: 'vprops',
+    data: 'vdata',
+    on: 'vOn'
```

Comment:
`v-on` is different from `vprops` and it will break the consistency of name convention.
But I think `v-*` looks a better convension...
em... how about making`vprops` deprecated and renamed to `v-props`?

Reply:
Oh, this is not a ‘request changes’ but my personal thought. You don’t need to change this pr because of this

Reply:
I think that's a great idea. If we go a bit further, how about renaming `vprops` to `v-bind`?

Reply:
I like `vprops` as is since it maps nicely to `props: {}` in the Vue component. Changing it to`v-props` would be enough, isn't it?

Reply:
Fair enough! 😉

Reply:
Changes are pushed in PR https://github.com/ngVue/ngVue/pull/53. Please, do review 😉